TNL-6794: adds enable auto auth jobs for ora2

jlajoie opened the pull request:
## [TNL-6794](https://openedx.atlassian.net/browse/TNL-6794)

### Description

Adds a job to set AutoAuth flag to true in the lms/cms config files and restart edxapp.

Should be merged in the same time as https://github.com/edx/jenkins-job-dsl-internal/pull/84
**FYI**
@staubina 
@cahrens

jlajoie commented:
@edx/devops this should be ready for review

jibsheet commented on ora2/resources/enable-auto-auth.sh:
```diff
@@ -0,0 +1,3 @@
+# Install a test course
```
Copy/Pasta comment

jibsheet commented on ora2/jobs/EnableAutoAuth.groovy:
```diff
@@ -0,0 +1,41 @@
+class EnableAutoAuth {
+
+    public static def job = { dslFactory, extraVars ->
+        return dslFactory.job(extraVars.get('FOLDER_NAME', 'ora2') + '/enable-auto-auth') {
```
Please bring in common_logrotator and common_wrappers so that we rotate logs and add our standard console output changes.

jlajoie commented:
@jibsheet this should be good for another look when you have a chance, addressed the feedback for the comment, logrotators and wrappers, and squashed commits
